woocommerce/woocommerce-android
# [UI Tests] Enable `productListShowsAllProducts`.

### Why
`productListShowsAllProducts` was failing and then disabled on CI.

### How
Three changes were needed:
1. In the Products list, the product card now contains not only the stock status, but also the price:

<img width="327" alt="Screenshot 2022-08-24 at 17 43 34" src="https://user-images.githubusercontent.com/73365754/186448828-320efbe8-6803-4d02-9e6f-6d49d3a3bc1e.png">

So the Espresso matcher was updated.

2. There was no mock file for one of the recently introduced `.../experiments/...` requests (see #7052 and #7097)
3. On FTL, the `Product type` key/value on Product Details screen was outside the visible area, so required scrolling into it.

The commits history is a mess, but I decided to keep it as it is, for the history. At the state of the first commit, the test was running with no issues at all locally, so all further troubleshooting was caused by CI specifics.

### Testing instructions
- You can run the test locally and see it 🟢 
- You can also see that `productListShowsAllProducts` is [passing](https://console.firebase.google.com/u/1/project/api-project-108380595987/testlab/histories/bh.edfd947f2636efe3/matrices/5048276431535770555/details?stepId=bs.b35f30eb6869e8d2&testCaseId=40) on FTL.
